package.json

```diff
@@ -111,7 +112,6 @@
     "react-native-gesture-handler": "~2.16.1",
     "react-native-get-random-values": "~1.11.0",
     "react-native-gifted-chat": "^2.0.1",
-    "react-native-map-clustering": "^3.4.2",
     "react-native-maps": "1.14.0",
```

Comment:
shouldn't this be removed now?

Reply:
you're right, thanks
will be removed in the upcoming commit